guides/_subsections/common/card-catalog.md

```diff
@@ -1,9 +1,9 @@
 {% capture card-catalog-include-text %}
- The <i>Dashboard Card Catalog</i> contains a list of available cards that can be added to the dashboard. Clicking on a card will open a panel containing a preview.
```

Comment:
<i>Dashboard Card Catalog</i> is italicized here, but in lines 5 and 6, the same catalog is referenced as <code>Card</code> catalog.